src/main/java/duke/command/TodoCommand.java

```diff
@@ -0,0 +1,24 @@
+package duke.command;
+
+import duke.Ui;
+import duke.task.TaskList;
+import duke.task.Todo;
+
+public class TodoCommand implements Command {
+    public static final String COMMAND_WORD = "todo";
```

Comment:
Perhaps you can consider extracting out the string so it is easier to edit, but aside from that i like how your code is very neat